<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SonPageClass</class>
 <widget class="QWidget" name="SonPageClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1068</width>
    <height>594</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SonPage</string>
  </property>
  <widget class="QTextEdit" name="TargetTextHex">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>10</y>
     <width>361</width>
     <height>351</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Consolas</family>
    </font>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarPolicy::ScrollBarAlwaysOff</enum>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QTextEdit" name="TargetTextChar">
   <property name="geometry">
    <rect>
     <x>920</x>
     <y>10</y>
     <width>141</width>
     <height>351</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Consolas</family>
    </font>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarPolicy::ScrollBarAlwaysOn</enum>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QScrollArea" name="SourceScrollArea">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>538</width>
     <height>351</height>
    </rect>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarPolicy::ScrollBarAsNeeded</enum>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>536</width>
      <height>349</height>
     </rect>
    </property>
    <widget class="QTextEdit" name="SourceTextHex">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>351</width>
       <height>341</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Consolas</family>
      </font>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarPolicy::ScrollBarAlwaysOff</enum>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QTextEdit" name="SourceTextChar">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>0</y>
       <width>161</width>
       <height>341</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Consolas</family>
      </font>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarPolicy::ScrollBarAlwaysOn</enum>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QTreeView" name="TargetFieldTree">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>360</y>
     <width>501</width>
     <height>171</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
  </widget>
  <widget class="QTreeView" name="SourceFieldTree">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>360</y>
     <width>521</width>
     <height>171</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="MakeAssembly">
   <property name="geometry">
    <rect>
     <x>920</x>
     <y>540</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>MakeAssembly</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ShellcodeTamper">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>540</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>ShellcodeTamper</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
